### ISSUE TITLE
[BUG] `palette(:wong)` does not work after update to 1.2.1.

### ISSUE THREAD

**Issue by mmikhasenko** [JuliaPlots/Plots.jl#2669]
I used to do `color_patette = palette(:wong)` that does not work now.

Also, I often need to have the same color for several curves on the plot with a chosen `color_patette`. A relatively convenient way that I found was `palette(:wong)[i]`. This functionality seems to be deprecated. However, I do not find it mentioned in NEWS, Docs.

I thought [this discussion](https://github.com/JuliaPlots/Plots.jl/issues/2510) might be relevant, however, not sure when it changed.

### Versions

Plots.jl version:
```
Commit 46ce4d7933 (master of 2019-11-26 06:09 UTC)
Platform Info:
  OS: Windows (x86_64-w64-mingw32)
  CPU: Intel(R) Core(TM) i7-7500U CPU @ 2.70GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.1 (ORCJIT, skylake)
Environment:
  JULIA_NUM_THREADS = 2
```

**Comment by daschw** [comment]
Yes, that is now `theme_palette(:wong)` with PlotThemes 2.0. Note that you can also do `seriescolor = i` to choose the `i`th color of the plot's palette.

**Comment by mmikhasenko** [comment]
Wow, that is even better, thanks a lot!

It would be great to mention it in two places
 - [PlotThemes.jl/README.md](https://github.com/JuliaPlots/PlotThemes.jl/blob/master/README.md)
 - series attributes doc, ([generated?](https://github.com/JuliaPlots/PlotDocs.jl/blob/master/docs/make.jl#L32))

can be closed otherwise.
